feature/platform/transceiver/zr_low_power_mode_test/README.md

```diff
@@ -0,0 +1,99 @@
+# TRANSCEIVER-13: Configuration: 400ZR Transceiver Low Power Mode Setting.
+
+## Summary
+
+Validate 400ZR transceiver is able to move to low power consumption mode when
+the interface/config/enabled state is set to "False"
+
+**NOTE:** The Module Power Mode dictates the maximum electrical power that the
+module is permitted to consume while operating in that Module Power Mode.
+The Module Power Mode is a function of the state of the Module State Machine.
+Two Module Power Modes are defined:
+  * In Low Power Mode (characteristic of all MSM steady states except
+    ModuleReady) the maximum module power consumption is defined in the form
+    factor-specific hardware specification.
+  * In High Power Mode (characteristic of the MSM state ModuleReady) the
+    implementation dependent maximum module power consumption is advertised in
+    the MaxPower Byte 00h:201. More details in the CMIS link below.
+
+Link to CMIS:
+https://www.oiforum.com/wp-content/uploads/CMIS5p0_Third_Party_Spec.pdf
+
```

Comment:
Add a test topology section, for example: https://github.com/openconfig/featureprofiles/blob/main/doc/test-requirements-template.md?plain=1#L25-L27

Reply:
Done